source/Octopus.Tentacle.Contracts/Octopus.Tentacle.Contracts.csproj

```diff
@@ -28,7 +28,7 @@
     </Otherwise>
   </Choose>
   <ItemGroup>
-    <PackageReference Include="Halibut" Version="7.0.209" />
```

Comment:
do we also need to update the corresponding version in Server? I assume it's not required but possibly a nice tidy-up to do.

Reply:
Great point! I didn't think of Server but yes, I'd say we should remove the potential vulnerability there too by upgrading. I'll take care of that.